cmd/utils.go

```diff
@@ -137,18 +138,18 @@ func execContainer(containerName string, cmd []string) []byte {
 	}
 
 	defer connection.Close()
+
 	output, err := ioutil.ReadAll(connection.Reader)
-	if err != nil {
-		log.Fatal(err)
-	}
 
-	// Remove 8 first characters to get a readable content
-	// Sometimes the command returns nothing, without the following if the program fails without
-	// runtime error: slice bounds out of range
-	if len(output) > 0 {
-		return output[8:]
-	}
-	return nil
+	// Trim invalid characters
+	output = bytes.Trim(output, "\x01")
```

Comment:
@ErwanAliasr1 a bit better?

Reply:
Look at String Conversions  at https://golang.org/pkg/strconv/
So I think we need to convert instead of trimming some control bytes.